internal/infrastructure/db/migrations/20221020065955_integration_features_weight.sql

```diff
@@ -0,0 +1,36 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+ALTER TABLE integration_features ADD feature_weight float NULL;
+BEGIN;
+    UPDATE integration_features SET feature_weight = 1 WHERE feature_key = 'ev_battery';
```

Comment:
suggestion (non-blocking): For ease of scanning, something like [this fiddle](https://www.db-fiddle.com/f/8eRDvTna2JQrmJ9dX6tpkL/0) might be nice.